tui.py

```diff
@@ -1,232 +0,0 @@
-import collections
```

Comment:
Tell you what: if we make it an entire month into the beta without anyone egging our office in retaliation for bugs, I'll order you a custom commemorative t-shirt.

(the "t" stands for "tui")